src/scss/modal.scss

```diff
@@ -0,0 +1,18 @@
+.ReactModal__Overlay{
+  top: '50%';
+  left: '50%';
+  right: 'auto';
+  bottom: 'auto';
+  marginRight: '-50%';
```

Comment:
This looks like a sass-ism, do we have any conventions in place around using this vs `margin-right`?

Do these values all need to be quoted?